package.json

```diff
@@ -99,6 +99,7 @@
     "@tsconfig/recommended": "1.0.1",
     "@types/chrome": "0.0.237",
     "@types/classnames": "0.0.32",
+    "@types/content-disposition": "0.5.4",
```

Comment:
I can't find the library itself in `package.json`, should we explicitly add it?

Reply:
Did you run `make deps`?

Reply:
I mean in `package.json` in this PR (this file). I can only see this line change, which adds the type definitions, but not the library itself

Reply:
Ah, ok. It was already in the yarn.lock file (as a dependency of storybook, I think). I'll push a change to add it to package.json.

Reply:
thanks! yes, if we ever upgrade/remove storybook it might get removed or it's version change unexpectedly. If we're importing it it's best to explicitly call out the dependency.

I also wonder if that's why the webpack change was required (but I'm not sure about that one)

Reply:
> I also wonder if that's why the webpack change was required (but I'm not sure about that one)

Maybe? But I'm pretty sure I tried it at least once with both entries in the `package.json` and still got the webpack error (I added and removed everything a few times, manually edited package.json, threw stuff at the wall, etc. trying to sort it out lol).